src/tasks/override-links.js

```diff
@@ -10,6 +10,7 @@ var util = require('util'),
 
     REGEXP = {
     HREF: /<a\s+(?:[^>]*?\s+)?href="([^"]*)"/g,
+    SRC: /<img\s+(?:[^>]*?\s+)?src="([^"]*)"/g,
```

Comment:
It will be cool to have a comments with example match for this `regexp` line and others above.